Cards/Animation/CastPosition/CastPos_CentralPosition.gd

```diff
@@ -0,0 +1,30 @@
+extends CastPosition
+class_name CastPos_CentralPosition
+## Creates a single animation at a central position between all targets
+
+
+## If false, creates the animation at 0,0
+@export var use_central_position: bool = true
+@export var offset: Vector2 = Vector2.ZERO
+
+
+# @Override
+func initialize_animation(cast_animation_scene: PackedScene, list_targets: Array[Entity]) -> CastAnimation:
+	var cast_animation: CastAnimation = cast_animation_scene.instantiate()
+	
+	assert(list_targets.size() > 0, "Tried to create a CastPos_CentralPosition with no targets")
```

Comment:
why is this check made here and it is not made in CastPos_AllTargets ?
Should this be a push_error instead of an assert ? Since assert get removed when the game is compiled iirc

Reply:
Yeah true, i can change that